runtime/benches/accounts.rs

```diff
@@ -145,54 +150,50 @@ fn bench_delete_dependencies(bencher: &mut Bencher) {
 
 #[bench]
 #[ignore]
-fn bench_concurrent_read_write(bencher: &mut Bencher) {
```

Comment:
how about reverting back these nice AccountsDb-level bench tests? I think keeping and maintain these doesn't cost us much and might be handy to lay around the source tree in addition to the shinny direct DashMap/RwLock<HashMap> showdown. 

I guess, you just rewrote the body of these bench body code in a haste? ;)

Reply:
thanks, good catch!